t/full-stack.t

```diff
@@ -390,6 +389,14 @@ subtest 'Cache tests' => sub {
     $driver->get('/tests/8');
     ok wait_for_result_panel($driver, qr/Result: incomplete/), 'test 8 is incomplete';
 
+    subtest 'log shown within details tab (without page reload)' => sub {
+        $driver->find_element_by_link_text('Details')->click();
+        wait_for_ajax(msg => 'autoinst log embedded within "Details" tab');
+        ok(my $log = $driver->find_element('.embedded-logfile'), 'element for embedded logfile present');
+        wait_for_ajax(msg => 'log contents loaded');
+        like($log->get_text(), qr/Result: setup failure/, 'log contents present');
+    };
+
```

Comment:
Don't we have that covered in another test already? Do we really need that in the full stack test?

Reply:
> Don't we have that covered in another test already?

Unfortunately not. We only have it covered when the log file is already available at the time the page is loaded. To avoid the reload the file is loaded dynamically and that needs testing.

> Do we really need that in the full stack test?

This way I don't need to add an additional complex test setup (either complex because of all involved components or because of massive mocking). And adding the additional check to an already existing test seems beneficial for keeping the increased test execution time to a minimum. The test also already has checks for the autoinst log anyways so it doesn't seem to odd as well.

Reply:
yes, sure, makes sense. Just wanted to crosscheck.